plugin/src/main/java/org/opennms/timeseries/cortex/CortexTSS.java

```diff
@@ -412,10 +419,30 @@ public List<Metric> findMetrics(Collection<TagMatcher> tagMatchers, String clien
                 tagMatchersToQuery(tagMatchers));
         String json = makeCallToQueryApi(url, clientID);
         List<Metric> metrics = ResultMapper.fromSeriesQueryResult(json, kvStore);
-        metrics.forEach(m -> this.metricCache.put(m.getKey(), m));
+        asyncCacheSave(metrics);
```

Comment:
this change can be removed as we discussed, this wouldn't save any as this simple write in memory